src/fireblocks-sdk.ts

```diff
@@ -1123,6 +1125,65 @@ export class FireblocksSDK {
         return await this.apiClient.issueGetRequest("/v1/users");
     }
 
+    /**
+     * Gets all Console Users for your tenant
+     */
+    public async getConsoleUsers(): Promise<{ users: ConsoleUser[] }> {
+        return await this.apiClient.issueGetRequest("/v1/management/console-users");
+    }
+
+    /**
+     * Gets all Api Users for your tenant
+     */
+    public async getApiUsers(): Promise<{ users: ApiUser[] }> {
+        return await this.apiClient.issueGetRequest("/v1/management/api-users");
+    }
+
+    /**
+     * Create Console User for your tenant
+     * @param firstName firstName of the user, example: "Johnny".  Maximum length: 30 chars.
+     * @param lastName lastName of the user. Maximum length: 30 chars.
+     * @param email email of the user, example: "email@example.com"
+     * @param role role of the user, for example: "ADMIN"
+     * @param requestOptions
+     */
+    public async createConsoleUser(firstName: string, lastName: string, email: string, role: string, requestOptions?: RequestOptions): Promise<OperationSuccessResponse> {
+        const body = { firstName, lastName, email, role };
+        return await this.apiClient.issuePostRequest("/v1/management/console-users", body, requestOptions);
+    }
+
+    /**
+     * Create Api User for your tenant
+     * @param name name of the api user, example: "Johnny The Api".  Maximum length: 30 chars.
+     * @param role role of the user, for example: "ADMIN"
+     * @param csrPem generate .csr file and provide its string content here, example:  "-----BEGIN CERTIFICATE REQUEST-----aaa-----END CERTIFICATE REQUEST-----"
+     * You can find more info about csrPem and how to create it here: https://developers.fireblocks.com/docs/quickstart
+     * @param coSignerSetup your cosigner, for example: "SGX_MACHINE", read more: https://developers.fireblocks.com/docs/quickstart
+     * @param coSignerSetupIsFirstUser [SGX server enabled only] If you are the first user to be configured on this SGX-enabled Co-Signer server, this has to be true
+     */
+    public async createApiUser(name: string, role: string, csrPem: string, coSignerSetup?: string, coSignerSetupIsFirstUser?: boolean, requestOptions?: RequestOptions): Promise<OperationSuccessResponse> {
```

Comment:
Please use enums instead of strings. Also, it's better to pass a DTO instead of separated arguments for future compatibility (adding more properties)